src/cli/mod.rs

```diff
@@ -115,6 +115,16 @@ pub async fn execute() -> miette::Result<()> {
         )
     }))?;
 
+    // Honor FORCE_COLOR and NO_COLOR environment variables.
+    // Those take precedence over the CLI flag and PIXI_COLOR
+    let use_colors = match env::var("FORCE_COLOR") {
+        Ok(_) => true,
+        Err(_) => match env::var("NO_COLOR") {
+            Ok(_) => false,
+            Err(_) => use_colors,
+        },
+    };
+
```

Comment:
Let me know what you think about this logic.

Reply:
Looks great! Thanks

Reply:
Ok. I will mention it in the doc then.